tests/test_basic.py

```diff
@@ -62,3 +62,15 @@ def test_sftp_mkdir_rmdir_rename():
     lucky.rmdir(new)
 
     lucky.close()
+
+
+def test_ftp_session_already_gone():
+    lucky = Ska.ftp.SFTP('lucky', logger=logger)
+    # explicitly close paramiko channel, socket, and session
+    lucky.ftp.sock.get_transport().close()
+    lucky.ftp.sock.close()
+    lucky.ftp.close()
+    # delete ftp session
+    del lucky.ftp
```

Comment:
The original intent was to be able to `del lucky` on something that hasn't been nicely shut down along the way like this one is.  I guess this is an alternate test to make sure closing something it is already closed is OK.